src/js/angular/core/services/tracking/tracking.service.js

```diff
@@ -12,7 +12,8 @@ const modules = [
 
 ];
 angular.module('graphdb.framework.core.services.trackingService', modules)
-    .service('TrackingService', ['$window', '$jwtAuth', '$translate', '$licenseService', 'toastr', 'InstallationCookieService', 'GoogleAnalyticsCookieService', TrackingService]);
+    .service('TrackingService', ['$window', '$jwtAuth', '$translate', '$licenseService', 'toastr', 'InstallationCookieService', 'GoogleAnalyticsCookieService', 'LocalStorageAdapter', 'LSKeys',
+        TrackingService]);
```

Comment:
why the service is injecting itself?

Reply:
this is the service's function

Reply:
hmm I now noticed that you've used a service and not a factory.